src/screens/WhotoFollow/elements/ItemUser.js

```diff
@@ -6,7 +6,7 @@ import IconAdd from '../../../assets/icon/IconAdd';
 import IconCheck from '../../../assets/icon/IconCheck';
 import UserInfo from './UserInfo';
 import dimen from '../../../utils/dimen';
-import {colors} from '../../../utils/colors';
+import {COLORS} from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use the new `COLORS` object from the `theme` module instead of the old `colors` object. This change is consistent with the PR's goal of standardizing color usage across the application.
```diff
- import {colors} from '../../../utils/colors';
+ import {COLORS} from '../../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->